docs/source/reference/links.rst

```diff
@@ -126,6 +126,10 @@ VGG Networks
    chainer.links.VGG19Layers
    chainer.links.model.vision.vgg.prepare
 
+.. note::
+   ChainerCV contains implementation of VGG networks as well (i.e.,
+   :class:`chainercv.links.model.vgg.VGG16`).
```

Comment:
Could you clarify the difference from Chainer's one? Users may want to know which one is what they want.

Reply:
Thank you for the comment.